grammars/javascript.cson

```diff
@@ -1360,6 +1360,28 @@
         ]
       }
       {
+        'begin': '(?<=innerHTML\\s*(\\+?=)\\s*)(`)'
```

Comment:
Yeah. Aside: is there a way that I could set up a local testing environment so that I don't have to wait for Travis to finish each time I tweak something?